### ISSUE TITLE
'hanami secret' command

### ISSUE THREAD

**Issue by Phitherek** [hanami/hanami#625]
I would like to see a 'hanami secret' command - it would help with setting up session secrets in production environment. My proposal for this command is simple: just invoke SecureRandom.hex(32) as in generate app command and return the result to console.

**Comment by jodosha** [comment]
This is a convenient alternative to:

```shell
➜ ruby -rsecurerandom -e "puts SecureRandom.hex(32)"
b240729645bf9e54a85136f3c9524cac13c2862e8b84ae3cbb9942db52ff343b
```

**Comment by uaihebert** [comment]
Hey, I have created a PR with this implementation. I believe that is an useful and easy to create feature. ( :

**Comment by toch** [comment]
what about my PR previously submitted #627 ?

**Comment by uaihebert** [comment]
Ops, Did not notice your PR. Sorry.

**Comment by toch** [comment]
@uaihebert no worries. Maybe there is good thing we can merge together?

**Comment by uaihebert** [comment]
Mine approach is different from yours. I will close my pr. ( :

**Comment by jodosha** [comment]
Implemented by #627
